Makefile

```diff
@@ -225,6 +228,16 @@ ginkgo: $(GINKGO) ## Download ginkgo locally if necessary.
 $(GINKGO): $(LOCALBIN)
 	test -s $(LOCALBIN)/ginkgo || GOBIN=$(LOCALBIN) go install github.com/onsi/ginkgo/v2/ginkgo
 
+.PHONY: kuttl-test
+kuttl-test: ## Run kuttl tests
```

Comment:
This worked in my environment as documented with
`make kuttl-test KUTTL_ARGS="--test dataplane-deploy-no-nodes"`